android/library/src/main/res/values/colors.xml

```diff
@@ -36,4 +36,7 @@
     <color name="midGray">#8E8F94</color>
     <color name="lightRed">#E96F6F</color>
     <color name="orange">#F29853</color>
+
+    <color name="disabledTransparent">#88FFFFFF</color>
```

Comment:
FFFFFF is white, so we may call this opacity instead transparent. 
88 is the level of opacity. 
Based on this table link https://stackoverflow.com/questions/15852122/hex-transparency-in-colors?answertab=active#tab-top what you think to call this colors `opacity_50` for the opacity of 50% for example

Reply:
Alright. I changed the variables names to suggested pattern